capstone/capweb/templates/main_base.html

```diff
@@ -68,6 +68,12 @@
   </head>
   <body class="hamburger-menu-closed" tabindex="-1">
 
+    {% if SHOW_DEPRECATION_BANNER %}
+      <div class="deprecation-banner">
+        <strong>Deprecation Notice</strong>: As part of the process to transition the Caselaw Access Project dataset into a static state, new CAP accounts are no longer available and the CAP API will be sunset on September 1, 2024. You are currently viewing the legacy version of the Caselaw Access Project's website.
```

Comment:
```suggestion
        <strong>Deprecation Notice</strong>: As part of the transition of Caselaw Access Project data to a static state, new CAP accounts are no longer available and the CAP API will sunset on September 1, 2024. You are currently viewing the legacy version of the Caselaw Access Project's website.
```
Does this banner want a link to case.law?
I don't love "static state" but haven't thought of something better yet.

Reply:
Oh maybe no link for now, not until we cut over.